docker/Dockerfile

```diff
@@ -0,0 +1,48 @@
+ARG TAG=jammy
+FROM ubuntu:${TAG}
+
+ARG ROS_DISTRO
```

Comment:
Let's remove this since it's not necessary for this image

Reply:
Are you referring to removing lines 1-4 or only line 4?

Reply:
Just the `ROS_DISTRO` arg